### PR TITLE
doc: update signing device support & versions for taproot

### PR DESCRIPTION
**Pull request by pythcoiner**

### CODE REVIEW

File: doc/SIGNING_DEVICES.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ repository](https://github.com/wizardsardine/async-hwi).
 
 Version 1.5.0 and above of the firmware is supported for use in P2WSH descriptors.
 
-Support for use in Taproot descriptors has been implemented but not yet released.
+Support for use in Taproot descriptors you should use version 1.9.0 or higher.
```
Comment (jp1ac4) [inline doc/SIGNING_DEVICES.md:13]: I would remove "Support", i.e. "For use in Taproot descriptors...".

File: doc/SIGNING_DEVICES.md [hunk 2]
```diff
@@ -28,11 +28,8 @@ Support for use in Taproot descriptors is not yet available in the firmware.
 Support for use in both P2WSH and Taproot descriptors has only been released in Beta as of this
 writing. It is only supported by the [Edge
 firmware](https://github.com/Coldcard/firmware?tab=readme-ov-file#long-lived-branches).
+Support for use in Taproot descriptors you should use version 6.3.3 or higher.
```
Comment (jp1ac4) [inline doc/SIGNING_DEVICES.md:31]: Same as above to remove "Support".

**Review by jp1ac4 — COMMENTED** [review]
I've suggested a couple of minor changes.

**Comment by pythcoiner** [comment]
comments addressed

**Comment by edouardparis** [comment]
I think you should `git rebase -i master` and replace `pick` for `drop` the first three commits.

**Comment by pythcoiner** [comment]
done

**Review by jp1ac4 — APPROVED** [review]
ACK a94138ebb3ed8de92dea75f850fc0f573e7b4652.

**Pull request merged**